ludwig/schema/llms/peft.py

```diff
@@ -71,6 +71,10 @@ def to_config(self, **kwargs) -> "PeftConfig":
 @register_adapter(name="lora")
 @ludwig_dataclass
 class LoraConfig(BaseAdapterConfig):
+    def __post_init__(self):
```

Comment:
It might also be useful to log a warning in case alpha and r are both set if alpha isn't 2 * r since the best results tend to occur with 2 * r almost always based on experiments. But maybe that's not in scope for this PR.

Reply:
I personally think it's fine without a warning. If this becomes the default, hopefully people won't be changing it unless they know what they're doing. We may want to move its importance from 2 to 1, though.

Reply:
Makes sense!